src/Schema/Events/ImpressionTracking.ts

```diff
@@ -80,3 +80,28 @@ export interface TooltipViewed {
   context_owner_type: PageOwnerType
   type: string
 }
+
+/**
+ * A user sees a an error message
+ *
+ * This schema describes events sent to Segment from [[ErrorMessageViewed]].
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "errorMessageViewed",
+ *    context_owner_type: "orders",
+ *    context_owner_id: "order-id",
+ *    label: "This is an error",
+ *    error_code: "charge_authorization_failed"
+ *  }
+ * ```
+ *
+ */
+export interface ErrorMessageViewed {
+  action: ActionType.errorMessageViewed
+  context_owner_id: string
+  context_owner_type: OwnerType
+  label: string
+  error_code: string
+}
```

Comment:
The error dialog is displayed on several routes in the orders app in Force, so we would like to differentiate which route the error message is displayed on (ie. shipping, payment, counteroffer, etc.).

Could we have a separate field for the route?

Reply:
Should be on the context page path that we record always through the event!

Reply:
That works 👍